portal-web/test/functional/com/liferay/portalweb/macros/WorkflowMetrics.macro

```diff
@@ -81,4 +107,194 @@ definition {
 
 	}
 
+	macro verifyDefaultUnitsForEachDateRange {
+		var key_completionVelocityDateRange = "${completionVelocityDateRange}";
+
+		Click(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_DATE_RANGE_DROPDOWN");
+
+		Click(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_DATE_RANGE_DROPDOWN_OPTION");
+
+		if ("${key_completionVelocityDateRange}" == "Today") {
+			AssertElementPresent(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN_OPTION_INST_HOUR");
+
+			AssertTextEquals(
+				locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_CHART_UNIT_LABEL",
+				value1 = "inst / hour");
+
+			WorkflowMetrics.AssertChartIsPresent(chartVelocityUnit = "Inst / Hour");
+
+			Click(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN");
+
+			AssertElementNotPresent(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN_OPTION_INST_DAY");
+
+			AssertElementNotPresent(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN_OPTION_INST_WEEK");
+
+			AssertElementNotPresent(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN_OPTION_INST_MONTH");
+
+			AssertElementNotPresent(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN_OPTION_INST_YEAR");
+		}
+
+		if ("${key_completionVelocityDateRange}" == "Yesterday") {
+			AssertElementPresent(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN_OPTION_INST_HOUR");
+
+			AssertTextEquals(
+				locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_CHART_UNIT_LABEL",
+				value1 = "inst / hour");
+
+			WorkflowMetrics.AssertChartIsPresent(chartVelocityUnit = "Inst / Hour");
+
+			Click(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN");
+
+			AssertElementNotPresent(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN_OPTION_INST_DAY");
+
+			AssertElementNotPresent(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN_OPTION_INST_WEEK");
+
+			AssertElementNotPresent(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN_OPTION_INST_MONTH");
+
+			AssertElementNotPresent(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN_OPTION_INST_YEAR");
+		}
```

Comment:
Hey @barrosdennys , are the content inside of  _Today_ and _Yesterday_ clauses the same? If they are the same, I suggest you to put them in a single macro to enable reuse.